src/components/DeleteNetworkDialog.vue

```diff
@@ -117,6 +117,7 @@ export default Vue.extend({
   watch: {
     dialog() {
       if (this.dialog) {
+        this.clear();
```

Comment:
Is there a reason the clear needs to be done here and after closing the dialog (i.e., down on line 138)?

Reply:
I've updated this to just clear here (where you've commented), which is when the message is reset. Good catch

I did this across all the deletion components in e3a1b89